xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
```

Comment:
Sorry I didn't understand the short hand (maybe put a comment in the schema that all things with `bt_` are managed by the beamtime object?)